contributing-guides/style-guide.md

```diff
@@ -104,40 +104,60 @@ Example:
 Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 ```
 
-## Token syntax
+## Placeholder syntax
 
-User-provided values should use the `{{token}}` syntax
+User-provided values should use the `{{placeholder}}` syntax
```

Comment:
I actually like the name `placeholder` better than `token`. 👍🏼

Reply:
It's definitively more descriptive, especially for new contributors. (Even tough `token` just sounds cooler)